src/games/rota/board.tsx

```diff
@@ -80,20 +80,33 @@ export class Board extends React.Component<IBoardProps, {}> {
     }
   }
 
+  _getBoard(boardSize = 100) {
```

Comment:
This seems to be an constant, so why make it a prop? Wouldn't it be better for it to just be a constant inside Field?

Reply:
nope. there is one call with 50.